### ISSUE TITLE
Missing dist on NPM

### ISSUE THREAD

**Issue by jenil** [hypermodules/entypo#22]
The published v2.2.0 doesn't have a `dist/sprite.svg`

```
Error: ENOENT: no such file or directory, open '.../node_modules/entypo/dist/sprite.svg'
    at Object.openSync (fs.js:457:3)
    at Object.readFileSync (fs.js:359:35)
    at main (.../node_modules/entypo/index.js:7:13)
```

**Comment by ungoldman** [comment]
I think a side effect of removing `.npmignore` in #20 is `dist` is no longer in the published tarball.

>  If you want to include something that is excluded by your .gitignore file, you can create an empty .npmignore file to override it. 

- https://docs.npmjs.com/misc/developers

> All files in the package directory are included if no local .gitignore or .npmignore file exists. If both files exist and a file is ignored by .gitignore but not by .npmignore then it will be included.

- https://docs.npmjs.com/cli/publish

**Comment by jenil** [comment]
@ungoldman can you also push it to NPM?

**Comment by ungoldman** [comment]
@jenil I tried to publish yesterday but ran into a npm maintenance window. It's published now (v2.2.1). I inspected the tarball and the dist folder and sprite.svg file are there now. Hope that resolves all your issues.
